.github/workflows/review.yml

```diff
@@ -27,6 +27,8 @@ jobs:
       run: sudo apt-get update && sudo apt-get install gcc-${{ matrix.gcc-version }} g++-${{ matrix.gcc-version }}
     - name: 'Install dependencies'
       run: sudo apt-get install pkg-config automake autoconf autoconf-archive make libsgutils2-dev libudev-dev libpci-dev check
+    - name: 'Export -Werror flag'
+      run: export CFLAGS=-Werror
```

Comment:
For now, we don't have any issue with WERROR, so there is no need yet to avoid this flag.
I would prefer to make script with compile command and extra exports instead of having it here defined for every action separately.
I would like to include Werror in every compilation. Please take care of printing compilation command in log. CC must be configurable